components/controls/buttons.css

```diff
@@ -54,11 +54,17 @@
 
 .button-primary {
   background: var(--darkBlue);
-  color: #fff;
+  color: var(--white);
 }
 
 .button-primary:hover {
   background: var(--veryDarkBlue);
+  border-color: var(--veryDarkBlue);
+}
+
+.button-primary.button-light:hover {
```

Comment:
So it looks like this with the button light class:
![image](https://user-images.githubusercontent.com/1725611/100595470-3908f300-32fb-11eb-9299-0888bbae20b7.png)

And without:
![image](https://user-images.githubusercontent.com/1725611/100595627-779ead80-32fb-11eb-947b-83c130f18721.png)

Reply:
looks good!